src/Domain/Structure/ElementId.hpp

```diff
@@ -154,6 +154,26 @@ template <size_t VolumeDim>
 bool operator!=(const ElementId<VolumeDim>& lhs,
                 const ElementId<VolumeDim>& rhs);
 
+template <size_t VolumeDim>
+bool operator<(const ElementId<VolumeDim>& lhs,
```

Comment:
Maybe document (either dox or C++) what the comparison is and that it's arbitrary without some deep motivation behind it? I worry that in 2 years we'll look at this and ponder why it's comparing in this specific way :)